packages/backend/src/rpc.rs

```diff
@@ -17,7 +17,7 @@ pub fn router() -> Router<AppState> {
         .handler(new_ref)
         .handler(get_doc)
         .handler(head_snapshot)
-        .handler(save_snapshot)
+        .handler(create_snapshot)
```

Comment:
I renamed this since we are no longer passing content to be saved. However it might make sense to revert to `save_snapshot`.

Reply:
Good point, thanks!